### DISCUSSION TITLE
Card Pioneers

### DISCUSSION THREAD

**Discussion by jtmuller5** [jtmuller5/The-HustleGPT-Challenge#30] [Ventures]
![image](https://user-images.githubusercontent.com/47997351/226369156-da7576dd-f0ea-46aa-849e-62664fa9a1c0.png)

Website: [Card Pioneers](https://cardpioneers.framer.website/) 
Tagline: Crafting immersive digital card games for all ages 
Twitter: [@CardPioneers](https://twitter.com/CardPioneers)

**Comment by cardpioneers** [comment]
![DARING](https://user-images.githubusercontent.com/128401966/226391484-4c0dd749-e48a-48bb-b692-88ab52fb01eb.png)
Logo design for Card Pioneers first product - Daring Dinosaurs has been created! CardGPT has named this cute little dinosaur under the spotlight Dino Dazzle as this name is catchy, easy to remember, and reflects the fun and engaging nature of the Daring Dinosaurs game  😂

**Reply by cardpioneers** [reply]
Website has been updated to: https://cardpioneers.framer.website/

**Reply by jtmuller5** [reply]
I updated it in the first post 👍

**Reply by cardpioneers** [reply]
![DARING (1)](https://user-images.githubusercontent.com/128401966/226638447-d2a273ce-699e-4ed8-b119-52018412751f.png)
Wording for logo has been updated!

**Comment by cardpioneers** [comment]
Huge progress has been made today as we have our first draft of the 17 cards of Daring Dinosaurs 🎉
![cardgpt-38](https://user-images.githubusercontent.com/128401966/226638117-1acfdf67-a118-4c8b-992a-dfe4d2d6c1c9.PNG)

**Reply by cardpioneers** [reply]
https://twitter.com/CardPioneers/status/1639061858668908544?s=20 Thread on twitter introducing game mechanics and 2 type of cards: Extinction Event and Dino Dome

**Comment by cardpioneers** [comment]
Join Card Pioneers's Daring Dinosaurs Digital Prototype Playtesting: https://cardpioneers.gumroad.com/l/daringdinosaurs
